src/rdkafka_cgrp.c

```diff
@@ -3137,7 +4274,7 @@ rd_kafka_cgrp_unsubscribe (rd_kafka_cgrp_t *rkcg, int leave_group) {
 
 	rd_kafka_dbg(rkcg->rkcg_rk, CGRP, "UNSUBSCRIBE",
 		     "Group \"%.*s\": unsubscribe from current %ssubscription "
-		     "of %d topics (leave group=%s, join state %s, v%"PRId32")",
+		     "of size %d (leave group=%s, join state %s, v%"PRId32")",
```

Comment:
but a subscription might contain regexes which map to more than one topic, so although it might be less ambiguous, it is less ambiguous at the expense of being wrong sometimes.